spec/javascripts/packs/extensible-componenets.spec.js

```diff
@@ -0,0 +1,75 @@
+describe('Extensible components', function() {
+  var cmp;
```

Comment:
q. why var vs const, does it not support babel?

Reply:
Tests are not taken care by webpack or anything, so that's why we can't use `const`, `=>` and any ES6 goodies :cry:

Reply:
Tests should be treated with same love and care as production code.

Good testing tools do the JavaScript transpilation for you, so you don't have to mess with your existing webpack configuration. When using Jest, just add `.babelrc` file and Jest will use that.

@martinpovolny @karelhala BTW, Jest includes Jasmine as its test runner.